hack/manifest-gen/assets/charts/eck/values.yaml

```diff
@@ -0,0 +1,92 @@
+  selectorLabels:
+    control-plane: elastic-operator
+
+  # Container image options.
+  image:
+    # Full path to the operator container image excluding the tag. The tag value is derived from the operator.version field.
+    repository: docker.elastic.co/eck/eck-operator
+    # Image pull policy for the container.
+    pullPolicy: IfNotPresent
+
+  # Resource and request limits for the operator pod.
+  resources:
+    limits:
+      cpu: 1
+      memory: 150Mi
+    requests:
+      cpu: 100m
+      memory: 50Mi
+
+# Optional configuration values
+config:
+  # Set to true to enable creating cluster resources (e.g. ClusterRoles). If set to false, only namespace-level resources will be created.
+  createClusterResources: true
+
+  # Set of namespaces for the operator to manage. Set to empty array to manage all namespaces.
+  managedNamespaces: []
+
+  # Operator log level. Valid values are:
+  # -2 == ERROR
+  # -1 == WARN
+  #  0 == INFO
+  #  1 and above == DEBUG
+  logVerbosity: 0
+
+  # Endpoint for exposing Prometheus metrics. Set to 0 to disable the endpoint.
+  metricsPort: 0
+
+  # Default container registry to be used by the operator when pulling Elastic Stack container images.
+  containerRegistry: docker.elastic.co
+
+  # Parallelism of the operator.
+  maxConcurrentReconciles: 3
+
+  # Experimental APM tracing configuration for the operator.
+  tracing:
+    enabled: false
+    config:
+      ELASTIC_APM_SERVER_URL: http://localhost:8200
+      ELASTIC_APM_SERVER_TIMEOUT: 30s
+
+  # Configuration for the self-signed CA used by the operator to generate certificates for Elastic Stack applications.
+  ca:
+    # Validity period for the CA certificate.
+    validity: 8760h
+    # How long before expiry should the CA certificate be renewed.
+    rotateBefore: 24h
+
+  # Configuration for certificates generated by the operator.
+  certificates:
+    # Validity period for issued certificates.
+    validity: 8760h
+    # How long before expiry should the certificates be renewed.
+    rotateBefore: 24h
+
+  # Enable RBAC enforcement for cross-namespace references between Elastic Stack applications.
+  refs:
+    enforceRBAC: false
+
+  webhook:
+    # Enable the webhook.
+    enabled: true
+    # Allow the operator to create and manage the certificates for the webhook.
+    manageCerts: true
+    # Location of the webhook certificates. Should only be overridden if config.webhook.manageCerts is false and you are providing your own webhook configuration.
+    certsDir: "/tmp/k8s-webhook-server/serving-certs"
+    # Name of the Kubernetes secret containing the webhook certificates. Should only be overridden if config.webhook.manageCerts is false and you are providing your own webhook configuration.
+    secret: elastic-webhook-server-cert
+    # Name of the webhook.
+    name: elastic-webhook.k8s.elastic.co
+    # Name of the service created for the webhook.
+    serviceName: elastic-webhook-server
```

Comment:
should this be `service` (since we have `secret` and not `secretName`)? Or the other way around 🙃

Reply:
It's the name of the service that would be created and not the actual service itself so I think `serviceName` is more appropriate.